src/guide/configure/genesis.md

```diff
@@ -1,19 +1,29 @@
 # Genesis Block
 
-The genesis block is the first block in your blockchain. It's never empty,
-even if `configs/peer/genesis.json` is. We recommend adding at least one
-more account to the genesis block.
-
-In our case, it was _alice_@wonderland, with the public key
-`ed01207233bfc89dcbd68c19fde6ce6158225298ec1131b6a130d1aeb454c1ab5183c0`.
-Think of it as the password used to "log in" as _alice_.
+The **genesis block** is the first block in your blockchain. It's never
+empty, even if `configs/peer/genesis.json` is. Here's an example:
 
 ::: details Genesis Block Example: alice@wonderland
 
 <<< @/guide/configure/configs/genesis.json
 
 :::
 
+The **genesis account** is specified in the
+[peer configuration](./peer-configuration.md#genesis) file,
+`configs/peer/config.json`. This is the account that will submit the
+genesis block. The genesis account is like a super user account that has
+elevated privileges, but only during the genesis round. It should be signed
+by one of the peers (has the public key of this peer).
```

Comment:
What has the key? the Genesis block? Please make this more clear

Reply:
rephrased